src/common/StackStringStream.h

```diff
@@ -44,6 +44,8 @@ class StackStringBuf : public std::basic_streambuf<char>
 
   void clear()
   {
+    static std::stringbuf const ss;
+    basic_streambuf<char>::operator=(ss);
```

Comment:
could you elaborate a little bit what we are trying to address here? and we can always reset the read area pointers and write area pointers using
```c++
setg(nullptr, nullptr, nullptr);
setp(nullptr, nullptr, nullptr);
```

Reply:
i think `basic_streambuf` offers a mini framework for manipulating the derived class's internal buffer(s). but currently we are basically using the minimum subset of it -- we always overflow the inserted buffer into the buffer. i am posting #25814 to address this.

Reply:
ACK, I'll drop this change then. What about the other?